java/daikon/chicory/Instrument.java

```diff
@@ -388,7 +407,7 @@ private InstructionList call_initNotify(
    * Instrument all the methods in a class. For each method, add instrumentation code at the entry
    * and at each return from the method. In addition, changes each return statement to first place
    * the value being returned into a local and then return. This allows us to work around the JDI
-   * deficiency of not being able to query return values.
+   * deficiency of not being able to query return values. (We now instrument throws as well.)
```

Comment:
Please don't write comments like this.  Years from now, people won't know when this comment was written or why it is relevant.  Please describe the code, not the code's history.